app/gui2/src/components/GraphEditor/widgets/WidgetPort.vue

```diff
@@ -153,6 +153,7 @@ export const widgetDefinition = defineWidget(WidgetInput.isAstOrPlaceholder, {
     :data-h="randSlice"
   >
     <NodeWidget :input="innerWidget" />
+    <div v-if="connected" class="edgeAttachment" />
```

Comment:
Nitpick: I would put `edgeAttachment` before sub-widget, so it will be under. That probably would require some css adjustments.

Reply:
Used pure CSS solution instead now. The attachment graphic will be visually below the sub-widget, because it has a `z-index: -1` style.